v2.1/range-merges.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Range Merges
+summary: To help improve performance, CockroachDB can automatically merge small ranges of data together.
+toc: true
+---
+
+To help improve your cluster's performance, CockroachDB can automatically merge small ranges of data together to form fewer, larger ranges. This can both improve query latency, as well as your cluster's survivability.
+
+{{site.data.alerts.callout_danger}}
+
+In CockroachDB 2.1, range merges are an [experimental feature](experimental-features.html). However, in CockroachDB 2.2, we expect full support for range merges.
+
+Only use this feature if instructed to do so by a CockroachDB engineer.
+
+{{site.data.alerts.end}}
+
+## Settings
+
+### Enable/disable range merges
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_merge.queue_enabled` to:
+
+- `true` to enable range merges
+- `false` to disable range merges
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.range_merge.queue_enabled = true;
+~~~
+
+#### When to enable range merges
+
+Range merges can provide performance improvements if you're working with a CockroachDB engineer and running CockroachDB 2.1.3 or newer.
+
+#### When to disable range merges
+
+Disable (or do not use) range merges if you:
+
+- Want to manually control range splits using [`SPLIT AT`](split-at.html)
+
+- Use CockroachDB 2.1.2 or older
+
+## How range merges work
+
+### Overview
+
+CockroachDB splits your cluster's data into many ranges (64MiB by default), which are defined by a range of keys they contain. For example, your cluster might have a range for customers whose ID are between `[1000, 2000)`. If that range grows beyond 64MiB of data, the range is split into two 32MiB ranges.
+
+However, as you delete data from your cluster, a range might contain far less data. Over the lifetime of a cluster, this could lead to a number of small ranges.
+
+To reduce the number of small ranges, your cluster can have any range below a certain threshold (16MiB by default) try to merge with its "right-hand neighbor", i.e. the range that starts where this range ends. Using our example above, this might be the range for customers whose ID are between `[2000, 3000)`.
+
+If the combined size of the small range and its neighbor is less than the maximum range size, the ranges merge into a single range. In our example, this would create a new range of keys `[1000, 3000)`.
+
+{{site.data.alerts.callout_info}}
+
+Ranges only attempt to merge with their right-hand neighbor. Ranges do not currently attempt to merge with left-hand neighbor (i.e. the range that ends where this range begins).
+
+{{site.data.alerts.end}}
+
+### Why range merges improve performance
+
+#### Query latency
+
```

Comment:
Suggestion: If you say "this creates the following issues" new list items can be added in future revisions without having to edit the preamble (old habit of mine).